esmvalcore/preprocessor/_multimodel.py

```diff
@@ -1,14 +1,10 @@
-"""multimodel statistics.
+"""Statistics across cubes.
 
-Functions for multi-model operations
-supports a multitude of multimodel statistics
-computations; the only requisite is the ingested
-cubes have (TIME-LAT-LON) or (TIME-PLEV-LAT-LON)
-dimensions; and obviously consistent units.
+This module contains functions to compute statistics across multiple cubes or
+products.
 
```

Comment:
not quite - across means one applies a certain operation (statistic) to each cube or product individually from a set of cubes or products, this is different, it's a multicube/multiproduct statistic, and in the lingo of a climate modeller - multimodel statistic. Let's not reinvent the wheel :grin:

Reply:
Okay, I'm not a native english speaker, hence the confusion with 'across'. What word do you use for applying a function like this? 'Over'?

The reason I want to get rid of multi-model, is that we will use it for ensemble stats as well (which could be the same model). Multi-model is just a specific case of the more general multi-cube operations.

Reply:
just use multi-cube/multi-product statistics?

Reply:
Done